lib/commands/workspace.js

```diff
@@ -152,10 +166,16 @@ module.exports = {
       type: 'boolean',
       default: true,
     });
+    yargs.option('fetch', {
+      describe: 'Fetch module names from Github and store in local cache. Must be used along with --modules in order to work.',
```

Comment:
> Must be used along with --modules in order to work

Would Yarg's [`.implies()`](https://github.com/yargs/yargs/blob/master/docs/api.md#impliesx-y) help here?

Reply:
Good suggestion! I'll try that!